docs-website/sphinx/requirements.txt

```diff
@@ -1,4 +1,4 @@
--e ../../metadata-ingestion[datahub-rest]
+-e ../../metadata-ingestion[datahub-rest,sql-parsing]
```

Comment:
What's this for?

Reply:
DataHubGraph has a lineage method that uses sqlglot, so this helps sphinx not complain

doesn't actually impact much though